haskell-src-exts.cabal

```diff
@@ -88,6 +88,7 @@ Test-Suite test
                        tasty >= 0.3,
                        tasty-smallcheck,
                        tasty-golden >= 2.2.2,
+                       tasty-hunit,
```

Comment:
Why did you need to add this dependency? Usually we must have quite a good reason to add a dependency but this might be ok as it is only for the tests. I would still like to understand why it is necessary.